dotnet/src/webdriver/Chromium/ChromiumDriver.cs

```diff
@@ -76,6 +83,8 @@ public ChromiumDriver(ChromiumDriverService service, ChromiumOptions options, Ti
             this.AddCustomChromeCommand(DeleteNetworkConditionsCommand, HttpCommandInfo.DeleteCommand, "/session/{sessionId}/chromium/network_conditions");
             this.AddCustomChromeCommand(SendChromeCommand, HttpCommandInfo.PostCommand, "/session/{sessionId}/chromium/send_command");
             this.AddCustomChromeCommand(SendChromeCommandWithResult, HttpCommandInfo.PostCommand, "/session/{sessionId}/chromium/send_command_and_get_result");
+            this.AddCustomChromeCommand(LaunchAppCommand, HttpCommandInfo.PostCommand, "/session/{sessionId}/chromium/launch_app");
+            this.AddCustomChromeCommand(Permissions, HttpCommandInfo.PostCommand, "/session/{sessionId}/permissions");
```

Comment:
Does Chromium not also provide a "get permissions" API? It might not, but if it does, not including it here seems an oversight.

Reply:
This one is at the root level. It's likely part of Permission spec, but Firefox doesn't seem to implement it, and Safari has its own Permission interface, so I thought it would be easiest to leave it in Chromium implementation (where we have it in all the other bindings) for now. Might need to move it to top level everywhere.

Reply:
Oh, you asked about a getter vs setter here. No, just the setter: https://chromium.googlesource.com/chromium/src/+/main/chrome/test/chromedriver/server/http_handler.cc#859

Java Tests we check permissions with:

```java
    Map<String, Object> result = (Map<String, Object>) ((JavascriptExecutor) driver).executeAsyncScript(
      "callback = arguments[arguments.length - 1];"
      + "callback(navigator.permissions.query({"
      + "name: arguments[0]"
      + "}));", permission);
    return result.get("state").toString();
```